src/BenchmarkDotNet/Environments/ProcessorBrandStringHelper.cs

```diff
@@ -37,12 +38,11 @@ public static string Prettify([NotNull] string processorName, double? frequency
         /// Presents actual processor's frequency into brand string format
         /// </summary>
         /// <param name="frequency"></param>
-        private static string GetBrandStyledActualFrequency(double? frequency)
+        private static string GetBrandStyledActualFrequency(Frequency? frequency)
         {
             if (frequency == null)
                 return null;
-            var a = frequency / 1000;
-            return $"{a.Value.ToString("N2").Replace(',','.')}GHz";
+            return $"{frequency.Value.ToGHz().ToString("N2").Replace(',','.')}GHz";
```

Comment:
It will be better to use `ToStr()` (which use correct culture) instead of ugly `.Replace(',','.')`